src/main/webapp/dataUseLetter/DataUseLetter.js

```diff
@@ -10,15 +9,14 @@ import { InputFieldCheckbox } from '../components/InputFieldCheckbox';
 import { InputFieldTextArea } from '../components/InputFieldTextArea';
 import { AlertMessage } from '../components/AlertMessage';
 import { ConsentGroup, DUL } from "../util/ajax";
-import { Spinner } from '../components/Spinner';
-import { spinnerService } from "../util/spinner-service";
 import { MultiSelect } from "../components/MultiSelect";
 import { Search } from "../util/ajax";
 import { DataUse } from "../util/ajax";
 import _ from 'lodash';
 import { isEmpty } from "../util/Utils";
+import LoadingWrapper from "../components/LoadingWrapper";
 
-export const DataUseLetter = hh(class DataUseLetter extends Component {
+const DataUseLetter = hh(class DataUseLetter extends Component{
```

Comment:
revert space removal

Reply:
Unaddressed.

Reply:
I've must have missed this. this space removal has been reverted now.